src/main/java/com/github/codedoctorde/itemmods/api/block/CustomBlockManager.java

```diff
@@ -99,14 +100,15 @@ public static Location stringToLocation(final String location) {
     /**
      * @param location    The location where the custom block will be placed!
      * @param blockConfig The block config for the custom block
+     * @param player      The player who is placing the block
```

Comment:
can we make the player optional? when we break the block with a command in the console we don't have a player.

Reply:
This methods is only called by CustomBlockListener.onCustomBlockBreakand CustomBlockListener.onCustomBlockEntityHit.

In both events we have: `event.getPlayer()` and `Player) event.getDamager()`, so the player is always NotNull. I can change it to Nullable (maybe for forward compatibility)

Reply:
Please do it :D